### PR TITLE
Fixes on table

### PR DESCRIPTION
**Pull request by zerbfra**
Fix to preserve order when changing data (eventually we can come up with a boolean).
Fix to make checked items reactive (with prop)

### CODE REVIEW

File: src/components/table/Table.vue [hunk 1]
```diff
@@ -125,14 +125,18 @@
                 default: 20
             },
             paginationSimple: Boolean,
-            html: Boolean
+            html: Boolean,
+            checkedRows: {
+                type: Array,
+                default: () => []
+            }
```
Comment (rafaelpimpa) [inline src/components/table/Table.vue]: I think the default should be just an empty array instead of a function.

**Comment by rafaelpimpa** [comment]
I just accepted another PR on this Table file, I think you should clone the repo again and make your changes, since the last PR changed a lot of things.

Also, no need to build, I'll do this before publishing :)

**Comment by rafaelpimpa** [comment]
I'm adding the ``updatingData`` right now to the Table myself, didn' realise it before, thanks :)

File: src/components/table/Table.vue (hunk 1 above)
Comment (rodymol123) [inline src/components/table/Table.vue]: Using just the [] or {} as default value on props results in warnings being thrown by Vue, this is the solution as described by Vue's creator: https://github.com/vuejs/vue/issues/1032

Comment (rafaelpimpa) [inline src/components/table/Table.vue]: Thanks for the heads up. Didn't know that.

**Comment by rafaelpimpa** [comment]
Closing this since next release will cover all the last thing it was proposed here.

**Pull request closed without merging**